ui/demo/components/Reader.tsx

```diff
@@ -59,22 +63,7 @@ export const Reader: React.FunctionComponent<RouteComponentProps> = () => {
           <Header pdfUrl={samplePdfUrl} />
           <DocumentWrapper className="reader__main" file={samplePdfUrl} inputRef={pdfContentRef}>
             <Outline parentRef={pdfContentRef} />
-            <div className="reader__page-list" ref={pdfScrollableRef}>
-              {Array.from({ length: numPages }).map((_, i) => (
-                <PageWrapper key={i} pageIndex={i}>
-                  <Overlay>
-                    <HighlightOverlayDemo pageIndex={i} />
-                    <TextHighlightDemo pageIndex={i} />
-                    <ScrollToDemo pageIndex={i} />
```

Comment:
Since scroll function and citation annotations are available, I think we can take down above demo components.